src/helper.rs

```diff
@@ -16,3 +17,35 @@ pub struct Node {
     pub udp_port: u16,
     pub node_id: Identifier,
 }
+
+pub mod testing {
+    use super::*;
+
+    pub fn mk_nodes() -> Vec<Node> {
+        // Should these nodes have different IP addresses?
+        let listen_addr = String::from("127.0.0.1").parse::<Ipv4Addr>().unwrap();
+        let port_start = 9000_u16;
+
+        let our_nodes: Vec<Node> = (0..5)
+            .into_iter()
+            .map(|i| mk_node(&listen_addr, port_start, i))
+            .collect();
+
+        our_nodes
```

Comment:
less code good

```suggestion
        (0..5)
            .into_iter()
            .map(|i| mk_node(&listen_addr, port_start, i))
            .collect()
```

Reply:
I ended up needing to use our_nodes here:  https://github.com/EchoAlice/my_kademlia/commit/c7aaf37239813e69b2f63df3de8dfef117b7fdad

Lmk what you think